### PR TITLE
feat(projects): Allow to create and edit tasks

### PR DESCRIPTION
**Pull request by aziiee**

### CODE REVIEW

File: app/projects/route.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    this._super(...args);
+
+    if (this.user.isSuperuser || this.user.isReviewer) {
+      controller.set("hasAccess", true);
+      controller.fetchProjectsOfUser.perform();
+    } else {
+      controller.set("hasAccess", false);
+    }
+  }
+});
```
Comment (fkm) [inline app/projects/route.js]: Why did you use a route?

File: app/projects/template.hbs [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  {{#if selectedProject}}
+    <h3>
+      <span class="header-left">
+        Tasks of {{selectedProject.name}}
+      </span>
+      <span class="header-right">
+        <button class="btn btn-primary" {{action (perform createTask)}} data-test-add-task>Add Task</button>
+      </span>
+    </h3>
+    <div style="height: 400px; margin-bottom: 20px">
```
Comment (fkm) [inline app/projects/template.hbs]: See if you can find/create a CSS class you can use for that.

File: app/projects/route.js (hunk 1 above)
Comment (aziiee) [reply app/projects/route.js]: @fkm fixed used `init()` inside the controller instead

File: app/projects/template.hbs (hunk 2 above)
Comment (aziiee) [reply app/projects/template.hbs]: @fkm fixed

**Pull request merged**